python/ray/tests/test_state_api.py

```diff
@@ -2464,6 +2464,7 @@ def verify():
 
         assert tasks[0]["placement_group_id"] == pg.id.hex()
         assert tasks[0]["worker_id"] == workers[0]["worker_id"]
+        assert tasks[0]["worker_pid"] == workers[0]["pid"]
```

Comment:
We should add a test for task APi?

Reply:
wdym? this is already testing the listing tasks?